qemu/tests/timerdevice_boot.py

```diff
@@ -75,19 +76,39 @@ def verify_timedrift(session, is_hardware=False):
         timerdevice_drift_threshold = float(params.get(
             "timerdevice_drift_threshold", 3))
 
-        time_type = "system" if not is_hardware else "harware"
+        time_type = "system" if not is_hardware else "hardware"
         error_context.context("Check the %s time on guest" % time_type,
                               logging.info)
         host_time, guest_time = utils_test.get_time(session, time_command,
                                                     time_filter_re,
                                                     time_format)
         if is_hardware:
             guest_time = get_hwtime(session)
-        drift = abs(float(host_time) - float(guest_time))
-        if drift > timerdevice_drift_threshold:
-            test.fail("The guest's %s time is different with"
-                      " host's system time. Host time: '%s', guest time:"
-                      " '%s'" % (time_type, host_time, guest_time))
+        if not base_time:
+            drift = abs(float(host_time) - float(guest_time))
+            if drift > timerdevice_drift_threshold:
```

Comment:
@sitoliu  If base_time is "2006-06-17", then guest_time should be "2006-06-17",  drift value is much larger than 3 mins.